File: j2k/testData/fileOrElement/switch/nestedSwitchFallback.java
//method
public String foo(int i, int j) {
    switch (i) {
        case 0:
            switch (j) {
                case 1:
                    return "0, 1";
                case 2:
                    return "0, 2";
            }
        case 1:
            return "1, x";
        default:
            return "x, x";
}

File: nj2k/testData/newJ2k/switch/nestedSwitchFallback.kt
fun foo(i: Int, j: Int): String {
    return when (i) {
        0 -> {
            when (j) {
                1 -> return "0, 1"
                2 -> return "0, 2"
            }
            "1, x"
        }
        1 -> "1, x"
        else -> "x, x"
    }
}